generator.ts

```diff
@@ -59,10 +60,10 @@ export async function generate(token: string, configYaml: string): Promise<Repor
                 // run as many reports as we can but fail action if any failed.
                 let failed = [];
                 try {
-                    let report = require(reportModule) as ProjectReport;
+                    let reportGenerator = require(reportModule) as ProjectReportGenerator;
```

Comment:
At first I thought this was the generator but I see it's the report builder.  Maybe call it ProjectReportBuilder to disambiguate?